deployment/template_deployment.yaml

```diff
@@ -63,10 +63,11 @@ spec:
           image: IMAGE
           imagePullPolicy: Always
           args: [
-            "--bq_dataset", "google.com:datcom-store-dev.dc_v3_clustered",
-            "--bt_table", "dc22",
+            "--bq_dataset", "BQ_DATASET",
+            # Prod using dc22 and staging using dc1. Update this when bt_table change.
```

Comment:
Can you create a file to record the bt dataset. We only update that file (and that is gold standard) and here reads from that.